core/lib/state/src/rocksdb/mod.rs

```diff
@@ -224,6 +224,10 @@ impl RocksdbStorageBuilder {
         }
     }
 
+    pub(crate) fn skip_synchronize(self) -> RocksdbStorage {
```

Comment:
There's already a `doc(hidden) build_unchecked()` method below with the same semantics; you can reuse it. Weirdly enough, it's not used anywhere in the codebase; maybe, it was used in some tests that were removed.